apps/artemis-web/libs/graphql.ts

```diff
@@ -167,7 +167,7 @@ export class QueryGenerator {
   private getBgpCountQuery() {
     let args = '';
     if (this.options.hasColumnFilter) {
-      let column = Object.keys(this.options.columnFilter)[0];
+      const column = Object.keys(this.options.columnFilter)[0];
```

Comment:
is GQL eventually protected somehow or not? What is protected exactly?

Reply:
No just the REST api